tests/modeltests/tst_disassemblyoutput.cpp

```diff
@@ -126,6 +126,7 @@ private slots:
         }
 
         file->write(text.toUtf8());
+        file->flush();
```

Comment:
why was this not detected on the CI so far?

Reply:
:shrug: it worked on older versions, so I guess ether the host or the ci container got an update which changed  how the write calls are handled